src/test/java/com/zufar/icedlatte/review/api/ProductAverageRatingProviderTest.java

```diff
@@ -52,7 +52,7 @@ void getRatingAndReviewStatsSuccessful() {
         var reviewsCount = 0;
 
         var expectedRatingMap = new RatingMap(1, 0, 0, 0, 0);
-        var expectedStats = new ProductReviewRatingStats(randomID, "0.0", reviewsCount, expectedRatingMap);
+        var expectedStats = new ProductReviewRatingStats(randomID, "0,0", reviewsCount, expectedRatingMap);
```

Comment:
This test now fails, is this change intentional?

Reply:
This change was intentional, because in my case String formattedAvgRating = String.format(%.1f", avgRating); was equal to "0,0". If I change this way: String formattedAvgRating = String.format(Locale.US, "%.1f", avgRating); it would be equal to "0.0". Is it ok to leave this way? 
```suggestion
        var expectedStats = new ProductReviewRatingStats(randomID, "0,0", reviewsCount, expectedRatingMap);
```